src/main/java/de/tum/in/www1/artemis/service/ResultService.java

```diff
@@ -313,6 +314,10 @@ public Result createNewManualResult(Result result, boolean isProgrammingExercise
         return savedResult;
     }
 
+    public Result createNewManualResult(Result result, boolean isProgrammingExerciseWithFeedback) {
```

Comment:
```suggestion
    public Result createNewRatedManualResult(Result result, boolean isProgrammingExerciseWithFeedback) {
```

Reply:
I renamed the method and updated all occurrences.